### PR TITLE
fix(libs/movex-core-util): Fixed linting error in core-types.ts

### PR DESCRIPTION
**Pull request by Abhishek-Mallick**
Status:
Closes: https://github.com/movesthatmatter/movex/issues/126

Fixed linting issues in `core-types.ts` in `libs-movex-core-util`

### CODE REVIEW

File: libs/movex-core-util/src/lib/core-types.ts [hunk 1]
```diff
@@ -202,7 +203,7 @@ export type GenericResourceOfType<TResourceType extends string> = Resource<
 
 export type GenericResourceType = GenericResource['type'];
 
-export type MovexClient<Info extends UnknownRecord = {}> = {
+export type MovexClient<Info extends UnknownRecord = Record<string, never>> = {
```
Comment (GabrielCTroia) [inline libs/movex-core-util/src/lib/core-types.ts:206]: Hey @Abhishek-Mallick, thank you for these changes. As you're already using `Record<string, never>` in 2 places here, and weit could most likely be used in other places as well, I suggest to create a new type `EmptyObject = Record<string, never>` and use that everywhere is needed!

**Review by GabrielCTroia — CHANGES_REQUESTED** [review]
Just one suggestions

**Comment by Abhishek-Mallick** [comment]
Was having issues commiting over the previous PR for which created a new PR: https://github.com/movesthatmatter/movex/pull/144 with all the mentioned changes

**Comment by GabrielCTroia** [comment]
> Was having issues commiting over the previous PR for which created a new PR: #152 with all the mentioned changes

Ok, I'm going to close this PR then!

**Pull request closed without merging**